src/main/java/org/opentripplanner/transit/model/site/StationBuilder.java

```diff
@@ -99,6 +101,15 @@ public StationBuilder withTimezone(ZoneId timezone) {
     return this;
   }
 
+  public boolean isTransfersNotAllowed() {
+    return transfersNotAllowed;
+  }
+
+  public StationBuilder withNoTransfers(boolean noTransfers) {
+    this.transfersNotAllowed = noTransfers;
+    return this;
+  }
```

Comment:
We should be consequent, `transfersNotAllowed` does not necessarily means `noTransfers` - Make sure you use the same term in all places - in tests as well.